node-manager/src/localstorage.rs

```diff
@@ -94,934 +97,20 @@ impl MutinyBrowserStorage {
         LocalStorage::delete(mnemonic_key);
     }
 
-    pub(crate) fn get_nodes() -> gloo_storage::Result<NodeStorage> {
+    pub(crate) fn get_nodes() -> Result<NodeStorage, MutinyStorageError> {
         let res: gloo_storage::Result<NodeStorage> = LocalStorage::get(nodes_key);
         match res {
             Ok(k) => Ok(k),
             Err(e) => match e {
                 StorageError::KeyNotFound(_) => Ok(NodeStorage {
                     nodes: HashMap::new(),
                 }),
-                _ => Err(e),
+                _ => Err(e)?,
```

Comment:
yeah pretty sure... it would be a gloo storage and we're trying to get it automagically converted to a MutinyError... could be more explicit with `with_context` or some other move

not always sure why we don't get automatic "from" errors because we do have a few of those implemented